internal/pkg/externalplugins/labelblocker/labelblocker.go

```diff
@@ -126,24 +148,45 @@ func handle(cfg *externalplugins.Configuration, ctx labelCtx, gc githubClient, l
 			continue
 		}
 
+		var operate string
+
 		// Undo the illegal operation.
 		if ctx.action == LabeledAction {
+			// Remove the label added illegally.
 			err := gc.RemoveLabel(owner, repo, ctx.number, ctx.label)
 
 			if err == nil {
-				log.Infof("Remove %s label added illegally", ctx.label)
+				operate = fmt.Sprintf("Remove %s label added illegally", ctx.label)
```

Comment:
I don't think it's necessary to use variables, the original way of writing is quite good. But it would be better to put a period in all logs.